src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -39,15 +39,15 @@
 
   private final String manifestFolder;
   private Map<Integer, String> precincts;
-  private Map<Integer, Contest> contests;
+  public Map<Integer, Contest> contests;
```

Comment:
Please keep private and make a project-visible getter (i.e. no need for `public` even on that).